cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -1272,8 +1272,41 @@ def test_deletion_policy_validator(deletion_policy, name, expected_message, fail
 
 @pytest.mark.parametrize(
     "avail_zones_mapping, cluster_subnet_cidr, are_all_security_groups_customized, security_groups, expected_message, "
-    "failure_level",
+    "failure_level, file_system_info",
```

Comment:
NIT: The expected message should be the last parameter (based on the structure of other tests)

Reply:
moved